src/components/Albums.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <div>
+        <h2>This is Dinos</h2>
+        <AlbumList
+        :albums="albums"
+        />
+    </div>
+</template>
+
+<script>
+import AlbumList from './AlbumList.vue';
+import albumApi from '../helper/albumApi.js';
```

Comment:
Traditionally, where your data & api lives would be in a services folder. Helper folders usually indicate other functions.  Don't worry about changing this for now though :)
```suggestion
import albumApi from '../services/albumApi.js';
```